lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -300,10 +300,11 @@ def bootstrap(proto=nil)
             exit 1
           end
 
-          chef_vault_handler.run(node_name: config[:chef_node_name]) if chef_vault_handler.doing_chef_vault?
-
           client_builder.run
           bootstrap_context.client_pem = client_builder.client_path
+
+          chef_vault_handler.run(node_name: config[:chef_node_name]) if chef_vault_handler.doing_chef_vault?
```

Comment:
Just to keep parity with chef core, we should put this between `client_builder.run` and `bootstrap_context.client_pem = client_builder.client_path`.

I don't think there are any functional ramifications there.